theme/static/style/style.css

```diff
@@ -397,6 +397,33 @@ pre code {
     padding-left: 0;
 }
 
+/* Community Page */
+
+.community {
+    padding-left: 10%;
+    padding-right: 10%;
+    line-height: 2.8rem;
```

Comment:
Remove line-height and make paddings 5%, add `box-sizing: border-box`. This makes the page look like the blog pages.

Also add this style, to make the first h1 align in such a way as to represent the blog titles.
```
.community div.col-12 {
    padding-top: 6%;
    line-height: 1.4em;
 }
```